rest_framework/schemas/inspectors.py

```diff
@@ -11,6 +11,7 @@
 from django.utils.translation import ugettext_lazy as _
 
 from rest_framework import serializers
+from rest_framework import exceptions
```

Comment:
Move this import one line up to resolve lint errors on python 2.7

Reply:
Thanks!